src/NuGet.Core/NuGet.Protocol.Core.Types/WebRequestEventArgs.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+
+namespace NuGet.Protocol.Core.Types
+{
+    public class WebRequestEventArgs : EventArgs
```

Comment:
That's why a started a refactoring branch for logging, as I don't want to pass on the logger through all these layers. Agreed we should get rid of the events for logging.